src/Sulu/Bundle/TrashBundle/Resources/config/lists/trash_items.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" ?>
+<list xmlns="http://schemas.sulu.io/list-builder/list">
+    <key>trash_items</key>
+
+    <joins name="user">
+        <join>
+            <entity-name>%sulu.model.user.class%</entity-name>
+            <field-name>%sulu.model.trash_item.class%.user</field-name>
+        </join>
+        <join>
+            <entity-name>%sulu.model.contact.class%</entity-name>
+            <field-name>%sulu.model.user.class%.contact</field-name>
+        </join>
+    </joins>
+
+    <joins name="translation">
+        <join>
+            <entity-name>%sulu.model.trash_item.class%_translation</entity-name>
+            <field-name>%sulu.model.trash_item.class%.translations</field-name>
+            <condition>%sulu.model.trash_item.class%_translation.locale = :locale</condition>
+        </join>
+    </joins>
+
+    <joins name="fallbackTranslation">
+        <join>
+            <entity-name>%sulu.model.trash_item.class%_fallbackTranslation</entity-name>
+            <field-name>%sulu.model.trash_item.class%.translations</field-name>
```

Comment:
doesnt this join multiple translation if the trash item has multiple translations? i think sucha a trash item would appear in the list multiple times? 🤔

Reply:
but it would be okay for me to merge it like this and think about it when implementing the functionality for a localized entity